src/overlays/actors/ovl_En_Gm/z_en_gm.h

```diff
@@ -6,9 +6,21 @@
 
 struct EnGm;
 
+typedef void (*EnGmActionFunc)(struct EnGm*, GlobalContext*);
+
 typedef struct EnGm {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x184];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[18];
+    /* 0x01FC */ Vec3s transitionDrawTable[18];
+    /* 0x0268 */ EnGmActionFunc actionFunc;
+    /* 0x026C */ EnGmActionFunc subActionFunc;
+    /* 0x0270 */ ColliderCylinder collider;
+    /* 0x02BC */ s8 objGmBankIndex;
+    /* 0x02BE */ s16 counter;
+    /* 0x02C0 */ s16 eyeTimer;
+    /* 0x02C2 */ u8 eyeIndex;
```

Comment:
Wouldn't it be better to use plural for "eye" everywhere? Medi-goron has two eyes iirc